### ISSUE TITLE
[BUG] jquery syntax hightlighting is broken

### ISSUE THREAD

**Issue by BlackCatReal** [notepad-plus-plus/notepad-plus-plus#15650]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

![syntax_broken](https://github.com/user-attachments/assets/c0c9e320-6946-4301-b7fb-f17b5dddcefa)
all as displayed on screenshot, hope for a quick fix

### Steps To Reproduce

1.  try this structure:
```javascript
`				$('#chat-box').append(`
					<div class="chat-message universe">
						<img class="message-avatar" src="/views/frontend/images/universe.png" alt="Вселенная">
						<div class="message-content">
							<p>${message}</p>
						</div>
					</div>`);
					
				$('#chat-box').scrollTop($('#chat-box')[0].scrollHeight);
			}, 3000); // Задержка 3 секунды перед появлением ответа`
```
### Current Behavior

jquery syntax is broken after brackets

### Expected Behavior

corrent jquery syntax highlight

### Debug Information

```shell
Notepad++ v8.6.9   (32-bit)
Build time : Jul 12 2024 - 05:08:26
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : "C:\Users\BlackCat\AppData\Local\Temp\_tcx\16\0\universe.a-devs.xyz\public_html\views\frontend\main.php"
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 7 Ultimate (64-bit)
OS Build : 7601.0
Current ANSI codepage : 1251
Plugins : 
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
```

**Comment by rdipardo** [comment]
Duplicate:

- #15368
- #15228 
- <https://github.com/notepad-plus-plus/notepad-plus-plus/issues/13946#issuecomment-2117302209>
- <https://community.notepad-plus-plus.org/post/96413>

**Comment by rdipardo** [comment]
Fixed upstream: ScintillaOrg/lexilla@3a78ad7
